google-http-client/src/main/java/com/google/api/client/util/OpenCensusUtils.java

```diff
@@ -42,26 +48,49 @@
 
   private static final Logger LOGGER = Logger.getLogger(OpenCensusUtils.class.getName());
 
+  /**
+   * Span name for tracing {@link HttpRequest#execute()}.
+   */
+  public static final String SPAN_NAME_HTTP_REQUEST_EXECUTE =
+      "Sent." + HttpRequest.class.getName() + ".execute";
+
   /**
    * OpenCensus tracing component.
    * When no OpenCensus implementation is provided, it will return a no-op tracer.
    */
-  static Tracer tracer = Tracing.getTracer();
+  private static Tracer tracer = Tracing.getTracer();
+
+  /**
+   * Sequence id generator for message event.
+   */
+  private static AtomicLong idGenerator = new AtomicLong();
```

Comment:
This should be per request not global. In an ideal world this should match with the message event on the server so better to start with 0 for each request.